sky/utils/kubernetes_enums.py

```diff
@@ -27,3 +27,24 @@ class KubernetesServiceType(enum.Enum):
     """Enum for the different types of services."""
     NODEPORT = 'NodePort'
     CLUSTERIP = 'ClusterIP'
+
+
+class KubernetesPortMode(enum.Enum):
+    """Enum for the different types of modes supported for opening
+    ports on Kubernetes.
+    """
+    INGRESS = 'ingress'
+    LOADBALANCER = 'loadbalancer'
+
+    @classmethod
+    def from_str(cls, mode: str) -> 'KubernetesPortMode':
```

Comment:
Can we just use `KubernetesPortMode(mode)` instead? The `enum.Enum` class implemented a `__call__` function to retrieve the corresponding enum from its value.

Reply:
Should be done.